plugins/strelaysrv/strelaysrv_

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env bash
```

Comment:
You are using no bash-specific features in this plugin (except for the irrelevant `function` keywords, which can be simply removed). Thus it would be good to use `/bin/sh` instead.